conf.yaml

```diff
@@ -180,7 +180,7 @@ contents:
           -
             title:      Azure Marketplace and Resource Manager (ARM) template
             prefix:     en/elastic-stack-deploy
-            current:    7.5
+            current:    7.6
```

Comment:
This branch of documentation doesn't exist yet and so it's causing this build err:
> 06:49:06 INFO:build_docs:Current branch <7.6> is not in <branches> in book <Azure Marketplace and Resource Manager (ARM) template> at lib/ES/Book.pm line 114.

Reply:
Thanks @lcawl 
Is there going to be an Azure Marketplace and Resource Manager (ARM) template branch? Do you know who I can ask? If there is, maybe I'll just leave it and retest later.

Reply:
Confirmed by @russcam - this trails the stack release. Set current to 7.5.